packages/ui-components/src/Avatar/Avatar.module.scss

```diff
@@ -0,0 +1,98 @@
+@import "../styles/tokens.scss";
+
+@mixin selectable-border($n) {
+  box-sizing: border-box;
+  border: 2px solid $n;
+}
+
+.avatar {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-shrink: 0;
+  font-family: Source Sans Pro, sans-serif; // TODO (koorosh): replace with fonts from tokes
+  font-style: normal;
+  font-weight: 600;
+  line-height: 1;
+  border-radius: 50%;
+  overflow: hidden;
+  user-select: none;
+}
+
+.intent-default {
+  background: $crl-neutral-2;
+  color: $crl-neutral-7;
+}
+
+.intent-info {
+  background: $crl-blue-1;
+  color: $crl-blue-3;
+}
+
+.intent-warning {
+  background: $crl-yellow-1;
+  color: $crl-yellow-4;
```

Comment:
Our warning color is `$crl-base-yellow`/`$crl-yellow-3`; comment re: base colors from above applies here too.

Reply:
Actually, according to the designs in [Figma](https://www.figma.com/file/B5AtEGdRbW3VxiBdg62TB8Xw/CRDB-Design-System_V2?node-id=72%3A0)  it is supposed to be `yellow-1` and `yellow-4` colors.

Probably the naming for `warning` and `danger` intents are misleading and have to be used the same as in designs: `invalid` and `pending`? What do you think?

Reply:
Okay, my apologies, It looks like there is some divergence from the Figma DS and what was encoded as tokens in the cloud repo at an earlier date (which implies I'll have to do some refactoring of that repo's tokens instead of vice versa).

That said, it looks like the warning colors are orange instead of yellow. 

Agreed that the intent labels would be better as `invalid` and `pending` as they reflect avatar-related states more accurately from a semantic standpoint.

Reply:
Done, names of intents are refactored to reflect the same names as in designs.